mmv1/templates/terraform/examples/dialogflow_version_full.tf.tmpl

```diff
@@ -0,0 +1,3 @@
+resource "google_dialogflow_version" "full_version" {
```

Comment:
Looks like this needs to depend on an agent existing in the project first. Lets change this to also use a new project similar to the update test.